leakcanary-android-core/build.gradle

```diff
@@ -9,7 +9,9 @@ dependencies {
 
   implementation deps.kotlin.stdlib
 
+  testImplementation deps.assertj_core
   testImplementation deps.junit
+  testImplementation deps.kotlin.reflect
```

Comment:
Fun fact: kotlin-reflect already was in the runtime, brought in by `mockito-kotlin`. So I just made it official

Reply:
As long as it's not in the LeakCanary exported deps, that's fine anyway.

Reply:
Yup, it's not in exported deps